lib/index.js

```diff
@@ -149,7 +151,7 @@ function getOptions(options) {
 
   options.error = function(err) {
     if (error) {
-      error(util._extend(new Error(), JSON.parse(err)));
+      error.call(options.context, util._extend(new Error(), JSON.parse(err)));
```

Comment:
Thanks and points for future consideration. :+1: 

TBH, I haven't delve into global environment, I cannot say for a fact whether it will land in libsass world or not. Maybe @xzyfer can confirm it.

However, I do think that it will not break the user code in this case even if global environment is introduced in libsass. This is because whatever libsass is sharing with node-sass, it arrives via the binding layer which uses v8 lib (via nan wrapper). That allows us to assign / inject any object to a `persistent` (v8 terminology) JS object. So this sharing with custom imports or functions is taken care by the binding code, where we can alter the state of other objects in the current loop (libuv's `uv_loop`).